react/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "PingOne for Customers Custom SignOn React Client App",
+  "name": "Create PingOne for Customers Custom SignOn React Client App Sample",
```

Comment:
PingCloud?

Reply:
@zcollier told not to use PingCloud, until we told officially otherwise.